gameobjects/units/cUnit.cpp

```diff
@@ -246,7 +246,7 @@ void cUnit::createExplosionParticle() {
         }
     }
 
-    if (iType == SIEGETANK || iType == DEVASTATOR && rnd(100) < 25) {
+    if ((iType == SIEGETANK || iType == DEVASTATOR) && rnd(100) < 25) {
```

Comment:
@stefanhendriks, I wonder if I put the parentheiss correctly. It seems this is what was intented, but it is different from what was executed. The randomness was only applied to the devastator. Do you know what this line should do?

Reply:
I think you found a bug here and correctly fixed it. In fact, I doubt the `SIEGEDIE` particle should appear for a `DEVASTATOR`, but alas, that was implemented so lets keep that for sure for now.